Header  header      # timestamp

uint32  intensity   # Intenisty Level
float64 distance    # Distance [m] measured by ToF